app/Http/Controllers/Store/BundleController.php

```diff
@@ -188,27 +244,33 @@ public function removeVoucherFromCurrentBundle(Registration $registration, Vouch
             $errors["foreign"] = [$voucher->code];
         }
 
-        return $this->redirectAfterRequest($errors, $registration);
+        // Back to manager in all cases
+        $successRoute = $failRoute = route(
+            'store.registration.voucher-manager',
+            ['registration' => $registration->id]
+        );
+
+        return $this->redirectAfterRequest($errors, $successRoute, $failRoute);
     }
 
     /**
      * Filters and prepares errors before returning to the voucher-manager
      *
      * @param $errors
-     * @param $registration
+     * @param $successRoute
+     * @param $failRoute
      * @return \Illuminate\Http\RedirectResponse
      */
-    public function redirectAfterRequest($errors, $registration)
+    public function redirectAfterRequest($errors, $successRoute, $failRoute)
     {
-        $route = route('store.registration.voucher-manager', ['registration' => $registration->id]);
         if (!empty($errors)) {
             // Assemble messages
             $messages = [];
             foreach ($errors as $type => $values) {
                 switch ($type) {
```

Comment:
so `errors["transactions"] == false` will still trigger this error message? That's a tad confusing.